src/main/java/com/example/bookstore/model/Order.java

```diff
@@ -0,0 +1,62 @@
+package com.example.bookstore.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Set;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "orders")
+@SQLDelete(sql = "UPDATE categories SET is_deleted = true WHERE id = ?")
```

Comment:
```suggestion
@SQLDelete(sql = "UPDATE orders SET is_deleted = true WHERE id = ?")
```
be attentive with copy-paste!